Renderer.swift

```diff
@@ -103,11 +104,16 @@ class Renderer: NSObject {
                 let projectionMatrix = ortho(left: 0, right: 200 * aspectRatio, bottom: 0, top: 200, near: -1, far: 10)
                 var uniforms = Uniforms(modelViewMatrix: mvMatrix, projectionMatrix: projectionMatrix)
                 
+                // REVIEW: I would look for a way that you can change the uniform values without having to re-create
+                // the whole command buffer every time.
```

Comment:
I think this notion is the biggest fruit, I just don't know how to make it happen, yet.